lib/main_containers.pm

```diff
@@ -77,7 +77,7 @@ sub load_image_tests_docker {
 
 sub load_host_tests_podman {
     my ($run_args) = @_;
-    if (is_leap('15.1+') || is_tumbleweed || is_sle("15-sp1+") || is_sle_micro || is_microos) {
+    if (is_leap('15.1+') || is_tumbleweed || is_sle("15-sp1+") || is_sle_micro || is_microos || is_leap_micro) {
```

Comment:
Hm...Doesn't it already mean run everywhere except of sle <= 15 ?

Reply:
done, and new VRs